kolibri_explore_plugin/jobs.py

```diff
@@ -94,13 +96,52 @@ def get_remotecontentimport_task(
     }
 
 
-def enqueue_task(task, user, queue=QUEUE, priority=Priority.HIGH, **params):
+def get_content_task_user():
+    """Get a Kolibri user for content task usage"""
+    return FacilityUser.objects.filter(
+        devicepermissions__can_manage_content=True
+    ).first()
+
+
+def enqueue_task(
+    task, user=None, queue=QUEUE, priority=Priority.HIGH, **params
+):
     """Enqueue task as new job"""
+    if user is None:
+        user = get_content_task_user()
     task_func = import_path_to_callable(task)
     job_data, _ = task_func.validate_job_data(user, params)
     return job_storage.enqueue_job(job_data, queue=queue, priority=priority)
 
 
+def enqueue_next_background_task():
+    """Locate the next background task and enqueue it for running"""
+
+    # If there's already a job in progress, do nothing. We only want a
+    # single background task running.
```

Comment:
Perhaps link to this issue, and include a brief rationale like “… only want a single background task running in order to not slow down the user interface and to leave room in the worker pool for user-triggered downloads”.

Reply:
That's a good idea. It's not really clear why you'd want to do that.